doc/guide/emacs.md

```diff
@@ -120,16 +122,16 @@ hacking in no time. All you have to do is to set the environment variables
   :config
   (require 'gambit
            (concat gambit
-             (if (equal "nixos" (system-name))
-               "/share/emacs/site-lisp/gambit.el"
-               "/misc/gambit.el")))
+                   (if (equal "nixos" (system-name))
```

Comment:
Emacs on NixOS doesn't return `"nixos"` for `(system-name)`, but the hostname. But that was wrong already. I'll have to somehow fix that in a further patch.

Reply:
This are the two other ways I found to check for Nix, but they are probably not 100% accurate.
``` lisp
(string-match "NixOS" (shell-command-to-string "uname -v")) ;; could fail on custom kernel
(getenv "NIX_PATH") ;; might detect systems that have Nix installed but didn't use it
```

Reply:
Ideally, NixOS would automagically configure all paths, though I admit I have no idea what that configuration should look like for Emacs.